benchmarks/tests/conftest.py

```diff
@@ -1,5 +1,7 @@
 import pytest
 
+pytest.register_assert_rewrite("benchmarks.tests._asserts")
```

Comment:
This is new to me. Why was it added?

Reply:
If you have modules that do `assert` and are not tests then they need to be registered to create proper assert messages.  Otherwise, instead of something like `Expected xy[z], Received xy[c]` you will get simply `AssertionError` without any clues as to what was actually different.

FWIW it was new to me as well.  I went down a bit of a rabbit hole yesterday learning about it.

Reply:
Oh, fantastic!!!! I need to read up on this and add this to Conbench too. Thanks!